src/webgpu/web_platform/canvas/readbackFromWebGPUCanvas.spec.ts

```diff
@@ -329,3 +329,33 @@ g.test('drawTo2DCanvas')
 
     readPixelsFrom2DCanvasAndCompare(t, ctx, expect[t.params.alphaMode]);
   });
+
+g.test('transferToImageBitmap_unconfigured_nonzero_size')
+  .desc(
+    `Regression test for a crash when calling transferImageBitmap on an unconfigured. Case where the canvas is not empty`
+  )
+  .fn(t => {
+    const canvas = createCanvas(t, 'offscreen', 2, 3);
+    canvas.getContext('webgpu');
+
+    // Transferring gives an ImageBitmap of the correct size filled with transparent black.
```

Comment:
If the `alphaMode` is `"opaque"` it should be opaque black - I think worth adding a test case

Reply:
How can there be an alphaMode since this has never been configured?

Reply:
Right. didn't think very hard about it when reading the spec, will check it and make sure the spec makes sense here